### PR TITLE
Update branding to 2.1.6-servicing

### PR DESCRIPTION
**Pull request by natemcmaster**
For consistency with other .NET Core teams, we'll use 2.1.x-servicing-buildnumber for servicing builds, not 2.1.x-rtm-buildnumber.

### CODE REVIEW

File: version.props [hunk 1]
```diff
@@ -27,8 +27,8 @@
     <VersionSuffix Condition=" '$(VersionSuffix)' != '' AND '$(BuildNumber)' != '' ">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
     <VersionSuffix Condition=" '$(VersionSuffix)' != '' AND '$(VersionMetadata)' != '' ">$(VersionSuffix)+$(VersionMetadata)</VersionSuffix>
 
-    <ExperimentalPackageVersion Condition=" '$(IsFinalBuild)' == 'true' AND '$(ExperimentalVersionSuffix)' == 'rtm' ">$(ExperimentalVersionPrefix)</ExperimentalPackageVersion>
-    <ExperimentalPackageVersion Condition=" '$(IsFinalBuild)' == 'true' AND '$(ExperimentalVersionSuffix)' != 'rtm' ">$(ExperimentalVersionPrefix)-$(ExperimentalVersionSuffix)-final</ExperimentalPackageVersion>
+    <ExperimentalPackageVersion Condition=" '$(IsFinalBuild)' == 'true' AND ('$(ExperimentalVersionSuffix)' == 'servicing' OR '$(ExperimentalVersionSuffix)' == 'rtm') ">$(ExperimentalVersionPrefix)</ExperimentalPackageVersion>
```
Comment (Eilon) [inline version.props]: Actually, 1 question: Why are these lines in the reverse order of the ones above for non-experimental ones?

**Comment by natemcmaster** [comment]
:up: :date:

image

File: version.props [hunk 2]
```diff
@@ -3,35 +3,43 @@
...
-    <PackageVersionSuffix Condition=" '$(IsFinalBuild)' != 'true' AND '$(BuildNumber)' != '' ">$(PackageVersionSuffix)-$(BuildNumber)</PackageVersionSuffix>
-    <PackageVersionSuffix Condition=" '$(IsFinalBuild)' == 'true' AND '$(PrereleaseVersionLabel)' != 'rtm' ">$(PrereleaseVersionLabel)-final</PackageVersionSuffix>
-    <!-- NB: VersionSuffix is empty if '$(IsFinalBuild)' == 'true' AND '$(PrereleaseVersionLabel)' == 'rtm' -->
+    <!-- Run the build with /p:IsFinalBuild=true to produce the product with 'final' branding and versioning -->
+    <IsFinalBuild Condition=" '$(IsFinalBuild)' == '' ">false</IsFinalBuild>
+    <IncludePreReleaseLabelInPackageVersion>true</IncludePreReleaseLabelInPackageVersion>
+    <IncludePreReleaseLabelInPackageVersion Condition=" '$(IsFinalBuild)' == 'true' AND ('$(PreReleaseLabel)' == 'servicing' OR '$(PreReleaseLabel)' == 'rtm')">false</IncludePreReleaseLabelInPackageVersion>
 
+    <!-- The 'human friendly' version to display in installers. In pre-release builds, this might be "2.0.7 Preview 2 Build 12356". In final builds, it should be -->
```
Comment (Eilon) [inline version.props]: Is this comment cut off?

Comment (natemcmaster) [reply version.props]: Got distracted by someone so I didn't finish my

Comment (Eilon) [reply version.props]: I hope it wasn't someone holding a nerf gun that was going to do

**Pull request merged**